src/MemoryKeyboard/controller.js

```diff
@@ -0,0 +1,71 @@
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from 'string-to-dom';
+
+export const MemoryButtons = {
+    PLUS: 'plus',
+    MINUS: 'minus',
+    RECALL: 'recall',
+    CLEAR: 'clear'
+};
+
+export class MemoryKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        let viewString = viewTemplate();
+        this.view = stringToDom(viewString);
+        this._num = 0;
+        this._bindEventMemoryPlus();
+        this._bindEventMemoryMinus();
+        this._bindEventMemoryRecall();
+        this._bindEventmemoryClear();
+    }
+
+    memoryPlus(value) {
```

Comment:
Do not mix order of public and private methods. First list all public then private.

You should put all bindings in a single function, there are too many methods otherwise.